queries/apex/highlights.scm

```diff
@@ -0,0 +1,340 @@
+[
+  "["
+  "]"
+  "{"
+  "}"
+  "("
+  ")"
+] @punctuation.bracket
+
+[
+  ";"
+  ";"
+  "."
+ ] @punctuation.delimiter
+
+;; Methods
+
+(method_declaration
+  name: (identifier) @method)
+
+(method_declaration
+  type: (type_identifier) @type)
+
+(method_invocation
+  name: (identifier) @method.call)
+
+(argument_list
+  (identifier) @variable)
+
+(super) @function.builtin
+
+(explicit_constructor_invocation
+  arguments: (argument_list
+    (identifier) @variable ))
+
+;; Annotations
+
+(annotation
+  name: (identifier) @attribute)
+
+(annotation_key_value
+  (identifier) @variable)
+
+
+;; Types
+
+(interface_declaration
+  name: (identifier) @type)
+(class_declaration
+  name: (identifier) @type)
+(class_declaration
+  (superclass) @storageclass) ;; 
+(enum_declaration
+  name: (identifier) @type)
+(enum_constant
+  name: (identifier) @variable);;
+
+(interfaces
+  (type_list
+    (type_identifier) @function ));;
```

Comment:
Why is this `@function`?

Reply:
I copied the original `highlights.scm` which was tailor made for vscode.
Some annotations are not used by nvim, I need to quickly replace them while I don't have good knowledge of what new annotation to use.. so, some might seem funny like this...

Reply:
> Why is this `@function`?

what's the matching annotation for `interface` ? Thank you so much for correcting these many. My purpose was to learn and fix along the journey :)

Reply:
It should be `@type`.